streams/src/test/java/org/apache/kafka/streams/integration/StreamsUncaughtExceptionHandlerIntegrationTest.java

```diff
@@ -199,6 +201,7 @@ public void shouldShutdownSingleThreadApplication() throws Exception {
 
     @Test
     public void shouldShutDownClientIfGlobalStreamThreadWantsToReplaceThread() throws Exception {
+        properties = basicProps();
```

Comment:
`properties` can be initialized in `setup`, right?

Reply:
You are right, there is no need to add basicProps() here, I'm not sure why I was doing this then...